presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -557,7 +557,7 @@ public StageId getStageId()
             return taskContext.getTaskId().getStageId();
         }
 
-        public Map<Symbol, Type> getTypes()
+        public TypeProvider getTypes()
```

Comment:
s/getTypes/getTypeProvider`?

Reply:
Type "types provider" is the entity describing the types, so "getTypes" is accurate. No need to name the method exactly after the return type.